pytest_inmanta/plugin.py

```diff
@@ -300,17 +303,22 @@ def project_factory(request: pytest.FixtureRequest) -> typing.Callable[[], "Proj
             "repo": repos,
             "modulepath": modulepath,
             "downloadpath": "libs",
-            "install_mode": install_mode,
+            "install_mode": install_mode.value,
         }
         yaml.dump(metadata, fd)
 
     ensure_current_module_install(os.path.join(test_project_dir, "libs"), in_place)
 
     def create_project(**kwargs: object):
+        load_plugins = not inm_no_load_plugins.resolve(request.config)
+
+        # If load_plugins is False (the option is not set), it might be because the old
```

Comment:
I wonder the same here: what if the user sets it to `False` explicitly? Is that even possible? The comment in `pytest_inmanta.parameters` seems to imply so but from the source I get the impression that it isn't possible.

Reply:
Yes, the reasoning wasn't complete here indeed.  This should be fixed with the new support for legacy options.